x11-wm/oroborus/oroborus-2.0.20-r1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Small and fast window manager"
+HOMEPAGE="http://www.oroborus.org"
+SRC_URI="mirror://debian/pool/main/o/${PN}/${PN}_${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="gnome"
+
+RDEPEND="x11-libs/libXpm
+	x11-libs/libXext
+	x11-libs/libSM
+	x11-libs/libICE
+	x11-libs/libX11"
+DEPEND="${RDEPEND}
+	x11-proto/xproto
+	x11-proto/xextproto"
+
+DOCS=( AUTHORS ChangeLog example.${PN}rc README TODO )
+
+src_configure() {
+	econf --disable-dependency-tracking
+}
```

Comment:
Should I also nix src_install and gnome use flag. I doubt that its useful anymore.